web/ui_utils.js

```diff
@@ -381,6 +381,10 @@ function noContextMenuHandler(evt) {
   evt.preventDefault();
 }
 
+function isFileSchema(url) {
+  return url.startsWith('file://');
```

Comment:
I'm not sure if you actually read *all* of https://github.com/mozilla/pdf.js/pull/9345#issuecomment-357431768, so please take a closer look at it! Quoting from it:
> Add a `isFileSchema` helper function similar to https://github.com/mozilla/pdf.js/blob/ba0a3aebd05fe85f5eef42260b1346d5856b6ba1/web/ui_utils.js#L384
> [...]
> 
> In this case, a solution not requiring a polyfill would be a much better choice here.

So, please just follow the *exact same* pattern as the `isDataSchema` function just below for this helper function and remove the polyfill.